IFC4x3/Quantities/w/Weight_2ueoe8QyrBduWuJHsi21zC/Documentation.md

```diff
@@ -0,0 +1 @@
+Total weight of object
```

Comment:
Add punctuation.

Reply:
this is a phantom quantity andf is will be deleted